### PR TITLE
added participants projects

### PR DESCRIPTION
**Pull request by aswanthkoleri**

### CODE REVIEW

**Comment by gauthamzz** [comment]
dude live link

File: projects.html [hunk 1]
```diff
@@ -109,7 +108,60 @@
                     {% endfor %}
                 </div>
             </section>
-            <!-- End of Projects Section -->
```
Comment (gauthamzz) [inline projects.html]: it is end of projects secion

add a comment end of projects section 

and add another comment start of student projects 

they should be seperate

**Comment by aswanthkoleri** [comment]
https://aswanthkoleri.github.io/opencodecollab/projects.html#projects

File: projects.html [hunk 2]
```diff
@@ -109,7 +108,60 @@
...
+                                    <div class="back">
+                                        <!-- back content -->
+                                        <div class="card stu-card">
+                                            <div class="avatar">
+                                                <p style="padding:10px;width: 200px;display: inline-block;margin-top: 9px;">
+                                                    <a style="text-decoration: none;" class="fa-2x" href="{{partproj.github}}" target="_tab">GitHub <i class="fa fa-external-link"></i></a>
+                                                </p>
+                                                <br><hr>
+                                                <center class="fa-2x">
+                                                    <i class="fa fa-code"></i> <span id="{{partproj.user}}-{{partproj.repo}}-language">HTML</span><br>
```
Comment (gauthamzz) [inline projects.html]: we dont need this number of starts, while clicking on the link it should redirect to the gh pages of that project

File: projects.html [hunk 3]
```diff
@@ -174,6 +226,15 @@ <h4 class="panel-title" style="text-align: left">{{project.name}} ({{project.sta
             {% for project in site.data.projects %}
             $('#{{project.user}}-{{project.repo}}-load-more button').click();
             {% endfor %}
+               $('#issues-accordion div a').first().click();
```
Comment (gauthamzz) [inline projects.html]: i dont think this is needed

File: _data/partproj.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+- name: Brick-pop
+  user: aswanthkoleri
+  repo: Brick-Pop
+  image: opencodecollab.jpg
+  github: https://github.com/aswanthkoleri/Brick-Pop/
```
Comment (gauthamzz) [inline _data/partproj.yml]: provide live link also

File: _data/partproj.yml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+- name: Brick-pop
+  user: aswanthkoleri
+  repo: Brick-Pop
```
Comment (gauthamzz) [inline _data/partproj.yml]: this field is not needed

File: _data/partproj.yml [hunk 6]
```diff
@@ -0,0 +1,6 @@
...
+  image: opencodecollab.jpg
```
Comment (gauthamzz) [inline _data/partproj.yml]: change the name of the image

File: _data/partproj.yml [hunk 7]
```diff
@@ -0,0 +1,6 @@
...
+  github: https://github.com/aswanthkoleri/Brick-Pop/
+  stack: html,css,js
```
Comment (gauthamzz) [inline _data/partproj.yml]: stack is not needed

**Comment by gauthamzz** [comment]
screenshot from 2017-02-17 17-41-08
its overlapping the sections fix that also

**Comment by gauthamzz** [comment]
still overlapping  and why rotate is there is only one info , no need to rotate . when clicked on image go to the gh page

**Comment by aswanthkoleri** [comment]
check whether its okay now

**Pull request closed without merging**